table/tables/partition.go

```diff
@@ -1260,3 +1260,8 @@ func (lt *ForRangePruning) compare(ith int, v int64, unsigned bool) int {
 	}
 	return -1
 }
+
+// Compare is for the purpose of exporting
+func (lt *ForRangePruning) Compare(ith int, v int64, unsigned bool) int {
```

Comment:
How about exposing the original function `compare` directly since there is no many references related to it?

Reply:
Done in the new commit.